theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
Actually, that @sk-yellow on the other hand... where is it used? Is @sk-gold replacing it?